server/service/src/sync/sync_status/test.rs

```diff
@@ -131,12 +131,23 @@ async fn sync_status() {
 /// * /final (manually called as last step)
 fn get_initialisation_sync_status_tester(service_provider: Arc<ServiceProvider>) -> Tester {
     Tester::new(service_provider.clone())
+        // 'site_status' is called by initialisation
+        .add_test("site_status", |ctx| {
```

Comment:
This was a bit tricky, i was never happy with this test, but it seemed like the only concrete way to validate sync statuses are correct.

Anyways now that we calling site_status during initialisation, this needs to be added to test, and also since it's called, previous_datetime is updated below so can't use it anymore for comparison.